share/man/man5/pf.conf.5

```diff
@@ -2151,19 +2151,19 @@ Only effective before the route lookup happened, i.e. when filtering inbound.
 .It Xo Ar divert-to Aq Ar host
 .Ar port Aq Ar port
 .Xc
-Used to redirect packets to a local socket bound to
-.Ar host
-and
+Used to
+.Xr divert 4
+packets to the given divert
 .Ar port .
-The packets will not be modified, so
-.Xr getsockname 2
-on the socket will return the original destination address of the packet.
+Historically OpenBSD pf has another meaning for this, and FreeBSD pf uses
+this syntax to support
+.Xr divert 4 instead. Hence,
+.Ar host
+has no meaning and can be set to anything like 127.0.0.1.
+If a packet is re-injected and does not change direction then it won't be
```

Comment:
igor complains about a contraction here (won't -> will not), not necessarily a requirement.